packages/core/src/services/ai/index.ts

```diff
@@ -171,6 +172,17 @@ export async function ai({
       experimental_output: useSchema
         ? Output.object({ schema: jsonSchema(schema) })
         : undefined,
+      experimental_download: async (
+        options: { url: URL; isUrlSupportedByModel: boolean }[],
+      ) => {
+        try {
+          return Promise.all(options.map(({ url }) => downloadWithCache(url)))
+        } catch (error) {
+          throw new Error(
```

Comment:
So one file failing fails all?

Reply:
yeah that's also how it works now i'm not changing behaviour here